engine/src/main/java/com/sandboni/core/engine/sta/connector/JsonConnector.java

```diff
@@ -1,84 +0,0 @@
-package com.sandboni.core.engine.sta.connector;
```

Comment:
Why is this not longer needed and the logic moved to HttpTemplate?

Reply:
One of the reasons is the parallel connectors execution